guides/SPI.md

```diff
@@ -18,9 +18,23 @@ Once installed and enabled on a site, this provider will be called to fetch appl
 
 ## Setup
 In order to become a _service provider_, you have two options:
-* Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
+* Register an application in the Wix Developers Center, and add the integration component that you wish to implement as described below. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+### Configure an integration component in the Development Center
+In order to enable Wix to communicate with your app, add configurations for your component:
+1. In the side menu under **Build your app**, click **Components** .
+2. In the upper right corner of the page, select **Add Component** > **Integration component**.
+3. Select the relevant component and click **Add Component**.
+4. In the JSON editor, configure the parameters by referencing the **Instructions** section on the right side of the page. For each parameter, add the parameter name and value in the JSON editor.
+
+    ![screenshot of json editor](../media/spi-config.png)  
+
+5. Click **Save**.
+
+> **Note:**
+> If you are working with a legacy SPI, configuration may not be via the Wix Developers Center. See instructions in the specific SPI. 
```

Comment:
Are there actually legacy SPIs? Who did you get this info from?

Reply:
@roysha-wix yes there are :)